examples/browser-verify/README.md

```diff
@@ -0,0 +1,48 @@
+## How to install
```

Comment:
I created a PR that converts your PR comments into a basic readme: https://github.com/risc0/risc0/pull/745

Reply:
You're the best!